src/handle_callback.ts

```diff
@@ -37,7 +38,7 @@ export async function handleCallback(
   const sessionId = crypto.randomUUID();
   const tokens = await oauth2Client.code.getToken(
     request.url,
-    oauthSession,
+    oauthSession ?? undefined,
```

Comment:
```suggestion
    oauthSession,
```
`oauthSession` is always defined, thanks to the previous assertion.